docs/images.coffee.md

```diff
@@ -21,7 +21,7 @@ Here is an example showing some of these options.
        .text('Proprotional to width', 0, 0)
      
     # Fit the image within the dimensions
-    doc.image('images/test.jpeg', 320, 15, fit: [100, 100])
+    doc.image('data:image/png;base64,iVBORw0K...', 320, 15, fit: [100, 100])
```

Comment:
Understood. I have reverted the changes to the example code and agree the mention above is most likely sufficient.